crates/wdk-build/src/cargo_make.rs

```diff
@@ -1007,6 +1008,51 @@ pub fn package_driver_flow_condition_script() -> anyhow::Result<()> {
     })
 }
 
+/// `cargo-make` condition script for `generate-certificate` task in
+/// [`rust-driver-makefile.toml`](../rust-driver-makefile.toml)
+///
+/// # Errors
+///
+/// This functions returns an error whenever it determines that the
+/// `generate-certificate` `cargo-make` task should be skipped. This only
+/// occurs when the `WdrLocalTestCert` cannot be added from the build directory
+/// to `WDRTestCertStore`.
+///
+/// # Panics
+///
+/// Panics if certmgr is unable to be run.
+pub fn generate_certificate_condition_script() -> anyhow::Result<()> {
+    condition_script(|| {
+        let output = Command::new("certmgr")
+            .args([
+                "-put",
+                "-s",
+                "WDRTestCertStore",
+                "-c",
+                "-n",
+                "WdrLocalTestCert",
+            ])
+            .arg(get_wdk_build_output_directory().join("WDRLocalTestCert.cer"))
+            .output()
+            .expect("Failed to run certmgr.exe.");
+
+        match output.status.code() {
+            Some(0) => Err(anyhow::anyhow!(
+                "WDRLocalTestCert already in WDRTestCertStore. Skipping certificate generation."
```

Comment:
```suggestion
                "WDRLocalTestCert found in WDRTestCertStore. Skipping certificate generation."
```
let's keep the existing error message